File: Python/Programmers/문자열다루기기본.py
def solution(s):
    # if len(s)==4 or len(s)==6:
    #     if s.isdecimal():
    #         return True
    #     else:
    #         return False
    # else:
    #     return False
    return s.isdecimal() and len(s) in (4,6)

print(solution("a234"))
print(solution("12345")) 
print(solution("1234"))

File: Java/Programmers/문자열다루기기본.java

public class 문자열다루기기본 {
	public static void main(String args[]) {
		Solution문자열다루기기본 s = new Solution문자열다루기기본();
		System.out.println(s.solution("0x16"));
	}
}

class Solution문자열다루기기본 {
	public boolean solution(String s) {
//		if(s.length() == 4 || s.length() == 6) {
//			try {
//				long num = Integer.parseInt(s);
//				return true;
//			} catch (Exception e) {
//			}	
//		}
//		return false;
		
		return s.matches("[0-9]{4}|[0-9]{6}");
	}
}